core/templates/I18nFooter.ts

```diff
@@ -50,7 +52,16 @@ angular.module('oppia').directive('i18nFooter', [
           };
           ctrl.$onInit = function() {
             ctrl.supportedSiteLanguages = SUPPORTED_SITE_LANGUAGES;
-
+            // Fetching the saved language code from cookie as $translate in
+            //  angularjs uses cookies to store the last used code and we need
+            // fetch that code in order to have constistency in translations in
+            // both angular and angularjs.
+            if (document.cookie.includes('NG_TRANSLATE_LANG_KEY=')) {
+              var lang = document.cookie.split('NG_TRANSLATE_LANG_KEY=')[1];
+              lang = lang.split(';')[0];
+              I18nLanguageCodeService.setI18nLanguageCode(
+                lang);
```

Comment:
Move to end of prev line. (You only need to break if, when not doing so, the call would span > 1 line.)

Reply:
Done!